From 99f63ade7bdc7810bd557656b98ce87ed77de94c Mon Sep 17 00:00:00 2001
From: Xiaosong Wei <xiaosong.wei@intel.com>
Date: Mon, 12 Aug 2013 22:37:29 +0800
Subject: Set output filter for all video codecs

It need to set output filter for all kind of video codecs so that render
can get correct resized yuv buffer.

HSD:5179207
Category: AOSP Enhancement
Domain: MEDIA
Upstream-Candidate: yes

Change-Id: Ic488178ae4862bbc9a30e75ddd9984e76bc90066
Signed-off-by: Xiaosong Wei <xiaosong.wei@intel.com>
---
 libvideoeditor/vss/src/M4VSS3GPP_Clip.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/libvideoeditor/vss/src/M4VSS3GPP_Clip.c b/libvideoeditor/vss/src/M4VSS3GPP_Clip.c
index 40612f3..bbbe1a6 100755
--- a/libvideoeditor/vss/src/M4VSS3GPP_Clip.c
+++ b/libvideoeditor/vss/src/M4VSS3GPP_Clip.c
@@ -646,8 +646,6 @@ M4OSA_ERR M4VSS3GPP_intClipOpen( M4VSS3GPP_ClipContext *pClipCtxt,
                 "M4VSS3GPP_intClipOpen: Vid dec started; pViDecCtxt=0x%x",
                 pClipCtxt->pViDecCtxt);
 
-            if( M4DA_StreamTypeVideoMpeg4Avc
-                == pClipCtxt->pVideoStream->m_basicProperties.m_streamType )
             {
                 FilterOption.m_pFilterFunction =
                     (M4OSA_Void *) &M4VIFI_ResizeBilinearYUV420toYUV420;
-- 
1.7.4.1

